### PR TITLE
feat(model): research output authors

### PR DESCRIPTION
**Pull request by jeysal**
For https://trello.com/c/dUrWu11Q/1230-show-authors-on-an-output-detail-page
No proper backend impl, just fixed type errors

### CODE REVIEW

**Comment by jeysal** [comment]
@johnytiago @peterstarling can someone add the proper backend impl for the field? :sweat_smile:

File: apps/asap-server/src/entities/research-output.ts [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+import {
+  GraphqlResearchOutput,
+  GraphqlTeam,
+  RestResearchOutput,
+} from '@asap-hub/squidex';
 
 import { parseDate } from '../utils/squidex';
+import { parseGraphQLUser } from './user';
 
 export const parseResearchOutput = (
```
Comment (johnytiago) [inline apps/asap-server/src/entities/research-output.ts:11]: Since this is no longer used, can we remove it?

**Review by johnytiago — APPROVED** [review]

File: apps/asap-server/src/entities/research-output.ts (hunk 1 above)
Comment (peterstarling) [reply apps/asap-server/src/entities/research-output.ts:11]: but we are:
https://github.com/yldio/asap-hub/blob/086f7d63072bd41cebe9fabf09db79041b64f84e/apps/asap-server/src/controllers/research-outputs.ts#L151

https://github.com/yldio/asap-hub/blob/086f7d63072bd41cebe9fabf09db79041b64f84e/apps/asap-server/src/entities/team.ts#L45

**Comment by peterstarling** [comment]
image
sorry @johnytiago I was going to remove my comment and accidentally deleted yours but it was a valid comment and i followed it here: https://github.com/yldio/asap-hub/pull/857/commits/ce33c53f89db79cb56edc549690f22983682b2ee

**Pull request merged**